Aptoide/aptoide-client-v8
# [MOB-470] Installed with aptoide using room

**What does this PR do?**

This PR aims at migrating the installed with aptoide feature from realm database to room.

**Database changed?**

   Yes 

**Where should the reviewer start?**

- [ ] AptoideInstallManager.java

**How should this be manually tested?**

Open aptoide, install an old version of an app. Go to apps, check that the update is there with a tag indicating that it was installed with Aptoide. Perform that update and check that it is converting in the ab test.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-470](https://aptoide.atlassian.net/browse/MOB-470)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
